.azure-pipelines/bazel.yml

```diff
@@ -72,6 +72,18 @@ steps:
       testRunTitle: "${{ parameters.ciTarget }}"
       searchFolder: $(Build.StagingDirectory)/tmp
     condition: always()
+  
+  - bash: |
+      sudo apt-get install gcc libpq-dev -y
+      sudo apt-get install python3-wheel python3-setuptools -y
+      pip3 install wheel slackclient
+      ./tools/process_xml.py
```

Comment:
Don't do sudo install here, you should be able to create virtual env and do pip3 install there, like https://github.com/envoyproxy/envoy/blob/master/docs/build.sh#L51

If gcc is needed, probably better to just run it inside docker (i.e. use ci/run_envoy_docker.sh).

Reply:
okay. let me try to use that.